src/main/java/com/openshift/internal/restclient/URLBuilder.java

```diff
@@ -223,8 +220,15 @@ private StringBuilder appendParameters(StringBuilder url) {
 	}
 
 	public URLBuilder watch() {
-		this.watch = true;
+		addParmeter("watch", "true");
 		return this;
 	}
 
+	public String websocket() {
+		String url = build().toString();
+		url = "wss" + url.substring(url.indexOf(":"));
+		LOG.debug(String.format("Built url: %s", url));
```

Comment:
LOG.debug("Built url: {}", url); 
so that .format() doesn't get evaluated if not in debug mode